scripts/setup/setup-reportstream.sh

```diff
@@ -40,9 +40,15 @@ yq eval '.[0].receivers[] |= (
     }
 )' -i "settings/STLTs/Flexion/flexion.yml"
 
+LOCAL_DOCKER_IMAGE_NAME=$(docker ps --filter "name=trusted-intermediary-router-1" | grep trusted-intermediary-router-1)
 echo "Updating local URL and host in transport settings..."
-sed -i '' "s|__TI_API_URL__|${TI_LCL_API_URL}|g" "settings/STLTs/Flexion/flexion.yml"
-sed -i '' "s|__TI_API_HOST__|$(extract_host_from_url "${TI_LCL_API_URL}")|g" "settings/STLTs/Flexion/flexion.yml"
+if [[ ! -z $LOCAL_DOCKER_IMAGE_NAME ]]; then
```

Comment:
Instead of adding an if statement here and adding a new env variable, I think we can update the existing one `TI_LCL_API_URL` to the docker local host name. You could add something like this to the top of the script
```
LOCAL_DOCKER_IMAGE_NAME=$(docker ps --filter "name=trusted-intermediary-router-1" | grep trusted-intermediary-router-1)
if [[ ! -z $LOCAL_DOCKER_IMAGE_NAME ]]; then
  export TI_LCL_API_URL="http://host.docker.internal:8080"
fi
```

Reply:
In that case, could we still set a variable at the top with the value of the TI endpoint depending if we're using docker or not? That would simplify and help avoid code duplication. Something like:
```
LOCAL_DOCKER_IMAGE_NAME=$(docker ps --filter "name=trusted-intermediary-router-1" | grep trusted-intermediary-router-1)
if [[ ! -z $LOCAL_DOCKER_IMAGE_NAME ]]; then
  ti_api_url=TI_LCL_API_URL
else
  ti_api_url=TI_DOCKER_LCL_API_URL_RS_CONFIG
fi
```